x-pack/plugins/monitoring/public/application/hooks/use_table.ts

```diff
@@ -136,33 +134,6 @@ export function useTable(storageKey: string) {
       sorting,
       pagination,
       onTableChange,
-      fetchMoreData: ({
```

Comment:
This used to have the async `updateData` function in here (https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/views/base_eui_table_controller.js#L131).  Now it does not and this already happens onTableChange